style.css

```diff
@@ -0,0 +1,88 @@
+body {
+	width: 100vw;
```

Comment:
please setup your IDE to use the single tabulation instead of double like now.  In HTML you are using single so you can follow the pattern in the entire project

Reply:
@KubaKlin You missed this comment, please take it into account

Reply:
i've changed it, please let me know if now its resolved properly

Reply:
Yes :)